fastapi_auth_api_key/dependencies.py

```diff
@@ -14,31 +14,27 @@
 
 from fastapi import Depends, status
 from fastapi.exceptions import HTTPException
-from fastapi.security import APIKeyHeader
-
-HTTP_API_KEY_HEADER = os.environ.get("FASTAPI_AUTH_HTTP_API_KEY_HEADER", "HTTP-API-KEY")
 
 
 def authenticated_auth_api_key(
-    key: Annotated[str, Depends(APIKeyHeader(name=HTTP_API_KEY_HEADER))],
     env: Annotated[Environment, Depends(odoo_env)],
     endpoint: Annotated[FastapiEndpoint, Depends(fastapi_endpoint)],
 ) -> AuthApiKey:
+    admin_env = Environment(env.cr, SUPERUSER_ID, {})
```

Comment:
Why do we need this `admin_env`?

Can't we just translate using the `env` or the `endpoint.env` ?